trinity/utils/version.py

```diff
@@ -16,3 +18,12 @@ def construct_trinity_client_identifier() -> str:
         # mypy Doesn't recognize the `sys` module as having an `implementation` attribute.
         imp=sys.implementation,  # type: ignore
     )
+
+
+def is_prerelease() -> bool:
+    try:
+        distro = pkg_resources.get_distribution("trinity")
+        # mypy thinks that parsed_version is a tuple. Ignored...
+        return distro.parsed_version.is_prerelease  # type: ignore
```

Comment:
Am I right that all Trinity versions that we currently put out are pre-releases (as in, have `alpha` suffix)? Doesn't that mean we slow down sync for everyone trying out Trinity at this stage?

Reply:
Ah wait, maybe I misread. This does not turn on `PYTHONASYNCIODEBUG` by default?

Reply:
> This does not turn on PYTHONASYNCIODEBUG by default?

Right, my first draft turned it on at first, but it was way too slow, maybe >10x slower syncing. It was annoying even in development, and made it hard to tell which things might improve performance.